build.gradle

```diff
@@ -47,6 +47,16 @@ task verify {
     integrationTest.mustRunAfter test
 }
```

Comment:
Just knowing that this exists has made the "Test on Commit" path viable:
![Screenshot 2022-09-28 at 11 32 00](https://user-images.githubusercontent.com/13031259/192757553-5fb6bf48-f61a-4109-a4cd-55d6622656ef.png)

Reply:
I wonder how "verify" got into this screen. This is not a standard Gradle task name.

Reply:
I like the Test on Commit idea.